packages/npm-vue/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Atom,
+  AtomMut,
+  Ctx,
+  CtxSpy,
+  Unsubscribe,
+  atom,
+  isAtom,
+  throwReatomError,
+} from '@reatom/core'
+// import { parseAtoms, type ParseAtoms } from '@reatom/lens'
+// import { ReadonlyDeep } from '@reatom/utils'
+import {
+  ref,
+  Ref,
+  onScopeDispose,
+  inject,
+  InjectionKey,
+} from '@vue/composition-api'
+
+export const reatomCtxKey = Symbol('reatomCtxKey') as InjectionKey<Ctx>
+
+export const reatomRef = ((target: any, ctx = inject(reatomCtxKey)!) => {
+  throwReatomError(
+    !ctx,
+    '"ctx" is not passed explicitly nor provided with "vue:provide"',
+  )
+
+  if (typeof target === 'function' && !isAtom(target)) target = atom(target)
+
+  const state = ref()
+  /**
+   * Whether target is a readonly (computed) atom
+   */
+  const ro = typeof target !== 'function'
+
+  let unsub: Unsubscribe
+  onScopeDispose(() => unsub?.())
+
+  return {
+    get value() {
+      unsub ??= ctx.subscribe(target, (atomState) => (state.value = atomState))
```

Comment:
Shouldn't we track a scope here? Is it possible that the value reading will appear out of the reactive scope?

Reply:
For vue it's a safe opertaion in any scope